1. Basic Web Development/server/dist/server.js

```diff
@@ -0,0 +1,21 @@
+"use strict";
```

Comment:
let's exclude compiled files in `.gitignore`

Reply:
I'll do that right now with both the frontend and backend